### DISCUSSION TITLE
Sort Pull Requests by Number of Changes

### DISCUSSION THREAD

**Discussion by rogerluan** [community/community#78897] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Either by line adds/removals or files changed. This will make it easier to run through all the minor changes before tackling major new feature pull-requests.

This would be a HUGE help to manage large projects, especially open source ones. It'd make it easier to find "low hanging fruits" for instance: PRs that only touch a few lines of code, quick to review, and potentially provide a lot of value. When these are hidden amongst tens or even hundreds of PRs, it's nearly impossible to find them.

----

<sub>I searched for an existing community discussion about this topic but couldn't find one, except https://github.com/isaacs/github/issues/353, so I'm creating this one. If there is one about this topic already, feel free to close this one and point me to the existing one 🙇 thank you!</sub>

**Comment by kengruven** [comment]
Related: #17275

**Reply by Avasam** [reply]
And https://github.com/orgs/community/discussions/21726
